extensions/skins/conversation_v1/Conversation.js

```diff
@@ -346,29 +351,36 @@ oppia.directive('conversationSkin', [function() {
         }, TIME_NUM_CARDS_CHANGE_MSEC + TIME_FADEIN_MSEC + TIME_PADDING_MSEC);
       };
 
-      var _addNewCard = function(stateName, contentHtml, interactionHtml) {
-        oppiaPlayerService.applyCachedParamUpdates();
-        $scope.transcript.push({
-          stateName: stateName,
-          contentHtml: contentHtml,
-          contentHtmlFocusLabel: focusService.generateFocusLabel(),
-          interactionHtml: interactionHtml,
-          interactionIsInline: oppiaPlayerService.isInteractionInline(
-            stateName),
-          interactionIsDisabled: false,
-          interactionInstructions: (
-            oppiaPlayerService.getInteractionInstructions(stateName)),
-          answerFeedbackPairs: []
-        });
+      $scope.getInteractionInstructions = function(stateName) {
+        return oppiaPlayerService.getInteractionInstructions(stateName);
+      };
+
+      $scope.isInteractionInline = function(stateName) {
```

Comment:
What's the purpose of removing the interactionIsInline property from the card, and checking it every time through oppiaPlayerService?